src/main/scala/io/moia/scalaHttpClient/HttpClient.scala

```diff
@@ -36,34 +41,34 @@ class HttpClient(
       headers: immutable.Seq[HttpHeader],
       deadline: Deadline,
       queryString: Option[String]
-  )(implicit executionContext: ExecutionContext, ctx: String = ""): Future[HttpClientResponse] =
+  )(implicit executionContext: ExecutionContext, ctx: NoLoggingContext): Future[HttpClientResponse] =
     super.request(method, entity, path, headers, deadline, queryString)
 }
 
 class LoggingHttpClient[LoggingContext](
```

Comment:
I... need more details about that feature 😅 
 so we have a HttpClient that extends from a LoggingHttpClient (because we need to log stuff in the middle).
This **LoggingHttpClient** could be of [NoLoggingContext] or [LoggingContext]. 
If LoggingContext, we use a logger passed as parameter (also to the httpLayer). If NoLoggingContext, we don't pass it as parameter but still have a logger: `Logger.takingImplicit(LoggerFactory.getLogger(getClass.getName))((msg: String, _: NoLoggingContext) => msg)` took from ..where?